wemake_python_styleguide/visitors/tokenize/statements.py

```diff
@@ -256,3 +259,196 @@ def _post_visit(self) -> None:
             self._check_individual_line(
                 line_tokens, previous_line_token, next_line_token,
             )
+
+
+@final
+@dataclass
+class InconsistentComprehensionContext(object):
+    r"""Context for individual bracket enclosures (i.e. [],\{\}, or ())."""
+
+    def __init__(self) -> None:
+        r"""
+        Initializes context for a given bracket enclosure (i.e. [],\{\}, or ()).
+
+        Helper class for InconsistentComprehensionVisitor which stores context
+        for the current (potential) comprehension we are in. Combined with a
+        stack to enable support for nested comprehensions.
+
+        _is_comprehension:
+        Flag is set if current clause is identified as a list comprehension.
+
+        seen_clause_in_line:
+        Flag is set when the current line already contains a clause, which
+        is either the action, each for loop, or the conditional. Starts off
+        as True to account for the action, which we don't actually visit.
+
+        seen_for:
```

Comment:
let's also update`seen_for` and all others in this docstring

Reply:
Fixed docstrings to reflect private variable names